src/librustdoc/html/templates/source.html

```diff
@@ -0,0 +1,19 @@
+<div class="example-wrap"> {# #}
+    <pre class="src-line-numbers">
+        {% for line in lines.clone() %}
```

Comment:
lines is `Range`, so it's free, but is clone required here?

Reply:
iirc, without that Askama tries to move it